app/controllers/resources_controller.rb

```diff
@@ -34,6 +34,10 @@ def push_to_onesky
     head :no_content
   end
 
+  def suggestions
+    render json: ToolFilterService.new(params).call, status: :ok
```

Comment:
I believe we should be able to include `include_param: params[:include], fields_param: field_params` when rendering this json to be able to support any includes/fields for the response

Reply:
I'm not totally sure if that will be beneficial or not

Reply:
ok, I'll work on it

Reply:
can you give me some curl request sample for this?

Reply:
```
https://mobile-content-api-stage.cru.org/resources/suggestions?filter[country]=AU&filter[confidence]=1&include=latest-translations&fields[resource]=name%2Cabbreviation%2Clatest-translations
```

Reply:
this is working correctly